### ISSUE TITLE
Narrator/NVDA not announcing anything when we are searching in the search workshop search box: A11y_Maow_Browse Workshop_Search Workshop_Info and Relationships

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/moaw#112]
### GitHub Tags
#A11yTCS; #A11ySev2; #ChromiumEdge; #WCAG1.3.1; #Win11; #GH_Maow_Web_Mar24; #DesktopWeb; #Maow; #Info and Relationships; #AILimited; #A11yMAS;

### Actual Result:
Narrator/NVDA not announcing anything when we are searching in the search workshop search box
### Expected Result:
Narrator/NVDA should announce the search result we are getting after searching in the search workshop search box.

### Environment Details:
Application Name: Maow
#URL: https://moaw.dev/catalog/
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
Microsoft Edge: Version 122.0.2365.92 (Official build) (64-bit)

### Repro Steps:

1. Hit the URL https://moaw.dev/
2. Moaw screen will get open.
3. TAB till "Browse workshops" control and press ENTER.
4. TAB till "Search Workshops" control.
5. Verify that Narrator/NVDA is announcing anything when we are searching in the search workshop search box or not.
### User Impact:
Screen reader will not know about the result user is getting after searching in the search box.

### Attachment:
**NVDA:**
![Presentation3](https://github.com/microsoft/moaw/assets/124244478/f4ba18d1-3eb8-4ac7-80db-0ed1724adb64)
**Narrator**
![Narrator](https://github.com/microsoft/moaw/assets/124244478/68b29b97-ffc1-4b17-a137-ca62747fafba)

**Comment by sinedied** [comment]
@AnKushSingh05 this one has been fixed, could you validate it?

**Comment by AnKushSingh05** [comment]
Verified the issue in below environment
https://moaw.dev/
Issue is fixed. Hence closing the bug

![Screenshot (1959)](https://github.com/microsoft/moaw/assets/124244478/58fecffc-65ac-4f80-a0bd-d3e8f9d6a652)
